Entities/AtomGroup.cpp

```diff
@@ -1908,24 +1870,25 @@ before adding them to the MovableMan.
 //                                       (m_Resolution ? m_Resolution : 1));
 //                float hiteeMassDenom = 0;
 
-                for (mapItr = hitMOAtoms.begin(); mapItr != hitMOAtoms.end(); ++mapItr)
+                for (map<MOID,deque<pair<Atom*,Vector>>>::value_type mapItr : hitMOAtoms)
```

Comment:
Same variable name issue here and in the for loop inside this one. Again, this one's a little trickier.

Reply:
I did `MOAtomMapEntry` here and in the other ones.
It looks futile to try to find descriptive names for these loop variables, they're too layered.

Reply:
Yeah, these ones are tough. I'm fine with `MOAtomMapEntry`, thanks for figuring something out.